test/compflags/lydia/library/exportArray/exportFuncWithArrayArg.h

```diff
@@ -0,0 +1,6 @@
+#include <stdint.h>
+
+#include "chpl-external-array.h"
+
+void foo(chpl_external_array* x);
+void chpl__init_exportFuncWithArrayArg(int64_t ln, int32_t fn);
```

Comment:
Is there an issue or something about the desire for the Chapel compiler to produce .h files for the exported stuff?

Reply:
Yeah, #9669

Reply:
It's going to be real nice when I get to implementing that